src/index.js

```diff
@@ -18,12 +22,21 @@ const getTabValue = path => {
     return null;
 };
 
+// prettier-ignore
+console.log(`Dropbox Key = ${process.env.REACT_APP_DROPBOX_KEY}, Secret=${process.env.REACT_APP_DROPBOX_SECRET}`);
```

Comment:
Lets remove unnecessary console.logs :) It fills up screen with unnecessary information. If you want to have debug information, then use some library which allows for logging to be enabled/disabled dynamically.